src/main/java/nl/ultimateapps/TechItEasy/service/Television.java

```diff
@@ -0,0 +1,21 @@
+package nl.ultimateapps.TechItEasy.service;
+
+public class Television {
+
+    private String name;
+
```

Comment:
Probeer aan een class ook altijd een id mee te geven zodat je in je controller dit id kan targetten inplaats van de index in een array.